### ISSUE TITLE
cannot load such file -- patron 

### ISSUE THREAD

**Issue by yegor256** [zold-io/zold#555]
```
zold node --nohup --invoice=912ecc24b32dbe74 --trace --memory-dump --host=b2.zold.io
/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:120:in `require': cannot load such file -- patron (LoadError)
	from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:120:in `require'
	from /var/lib/gems/2.3.0/gems/zold-0.16.25/lib/zold/http.rb:26:in `<top (required)>'
	from /var/lib/gems/2.3.0/gems/zold-0.16.25/lib/zold.rb:27:in `require_relative'
	from /var/lib/gems/2.3.0/gems/zold-0.16.25/lib/zold.rb:27:in `<top (required)>'
	from /var/lib/gems/2.3.0/gems/zold-0.16.25/bin/zold:32:in `require_relative'
	from /var/lib/gems/2.3.0/gems/zold-0.16.25/bin/zold:32:in `<top (required)>'
	from /usr/local/bin/zold:23:in `load'
	from /usr/local/bin/zold:23:in `<main>'
```

**Comment by yegor256** [comment]
@rultor release, tag is `0.16.26`

**Comment by yegor256** [comment]
fixed: 93fd3c4
